src/framework/theme/components/select/select.spec.ts

```diff
@@ -248,6 +249,30 @@ export class NbSelectWithFalsyOptionValuesComponent {
 })
 export class NbMultipleSelectWithFalsyOptionValuesComponent extends NbSelectWithFalsyOptionValuesComponent {}
 
+@Component({
+  template: `
+    <nb-layout>
+      <nb-layout-column>
+
+        <nb-select>
+          <nb-option-group [disabled]="optionGroupDisabled">
+            <nb-option [value]="1" [disabled]="optionDisabled">1</nb-option>
+          </nb-option-group>
+        </nb-select>
+
+      </nb-layout-column>
+    </nb-layout>
+  `,
+})
+export class NbOptionDisabledTestComponent {
+  optionGroupDisabled: boolean = false;
```

Comment:
Let's not specify the property type when we assign value during the declaration. I've described that slightly deeper in some comment above.